tests/utils.py

```diff
@@ -127,10 +124,28 @@ def get_test_tube_logger(save_dir, debug=True, version=None):
     return logger
 
 
-def load_model(exp, root_weights_dir, module_class=LightningTemplateModel):
+def get_data_path(expt_logger, path_dir=None):
+    # some calls contain only experiment not complete logger
+    expt = expt_logger.experiment if hasattr(expt_logger, 'experiment') else expt_logger
+    # each logger has to have these attributes
+    name, version = expt_logger.name, expt_logger.version
+    # only the test-tube experiment has such attribute
+    if hasattr(expt, 'get_data_path'):
+        return expt.get_data_path(name, version)
+    # the other experiments...
+    if not path_dir:
+        path_dir = ROOT_PATH
+    path_expt = os.path.join(path_dir, name, 'version_%s' % version)
+    # try if the new sub-folder exists, typical case for test-tube
+    if not os.path.isdir(path_expt):
+        path_expt = path_dir
+    return path_expt
```

Comment:
Lots of special cases for test tube here. Fine for now to get the build passing, but let's revisit later and see if we can avoid special cases.

Reply:
agree, this was just a side-work when I was updating paths/links and found that the master fails...